intellij-plugin/src/test/testData/migration/v3-to-v4/multiModule_after.gradle.kts

```diff
@@ -0,0 +1,23 @@
+dependencies {
+  implementation(project(":schema"))
+}
+
+apollo {
+  service("service1") {
+    packageName.set("com.example.service1")
+    dependsOn(project(":schema"))
+  }
+
+  service("service2") {
+    packageName.set("com.example.service2")
+    dependsOn(project(":schema"))
```

Comment:
The multi-module + multi-service is a bit weird because before you could have matadata that was used by only one service while now I think it's going to fail if it doesn't find a schema for the given service in the `:schema` module. There's no way to guess just by looking at the Gradle config of a single project though so this is fine for me 👍